packages/requisitions/src/types.ts

```diff
@@ -37,9 +37,25 @@ export interface SupplierRequisition extends Requisition {
   deleteLine?: (line: SupplierRequisitionLine) => void;
 }
 
+export interface CustomerRequisition extends Requisition {
+  isDeleted: boolean;
+  otherParty: Name;
+  lines: SupplierRequisitionLine[];
```

Comment:
should be `CustomerRequisitionLine` for `lines` and the mutations below

Reply:
thanks!